### PR TITLE
#257: ErrorResponse format description

### PR DESCRIPTION
**Pull request by iliax**

### CODE REVIEW

**Comment by iliax** [comment]
@workshur please review the format and I will add it as 4XX & 5XX responses to all endpoints after

File: kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml [hunk 1]
```diff
@@ -1058,6 +1058,37 @@ paths:
...
+        timestamp:
+          type: number
+          description: Response unix timestamp in ms
+        requestId:
+          type: string
+          description: Unique server-defined request id for convenient debugging
+        fieldErrors:
+          type: array
+          items:
+            $ref: '#/components/schemas/FieldError'
```
Comment (workshur) [inline kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml:1080]: `fieldErrors:` might be '#/components/schemas/FieldError' without any extra array wrappers
```
fieldErrors:
  $ref: '#/components/schemas/ContactInfo'
```

Comment (iliax) [reply kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml:1080]: @workshur we can have several fields validation errors, so this is why array is used 
example
```
  "fieldErrors": [
    {
      "fieldName": "name",
      "restriction": "must not be null"
    },
    {
      "fieldName": "partitions",
      "restriction": "should be > 0"
    },
  ]
```

Comment (workshur) [reply kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml:1080]: In this case, restriction filed should contain an array of messages. It is not critical for me.

Comment (iliax) [reply kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml:1080]: @workshur renamed fieldErrors to field**s**Errors to highlight that errors for several fields can be returned

**Review by workshur — APPROVED** [review]

**Review by germanosin — APPROVED** [review]

**Pull request merged**